storage/flux/window.go

```diff
@@ -16,17 +16,19 @@ import (
 // splitWindows will split a windowTable by creating a new table from each
 // row and modifying the group key to use the start and stop values from
 // that row.
-func splitWindows(ctx context.Context, in flux.Table, f func(t flux.Table) error) error {
+func splitWindows(ctx context.Context, in flux.Table, createEmpty bool, f func(t flux.Table) error) error {
```

Comment:
Can you change `createEmpty` to be `selector`? It seems like the conditional is whether the function is a selector or not and not whether create empty was used. If create empty is false, I think aggregates and selectors are the same. The main difference seems to be that aggregates will have empty tables with one row and selectors don't.